spec/features/application_record_creation_spec.rb

```diff
@@ -22,6 +22,7 @@
     end
     context 'application template has been marked as inactive' do
       it 'shows text explaining that the application is unavailable' do
+        application_template.position.update(not_hiring_text: 'custom text')
         application_template.update active: false
         visit current_url
```

Comment:
Changes look good to me as is - I do have one small suggestion

The setup code here seems to not be specific to the it block, so it may make sense to move it to a before :each within the context. This isn't necessary at the moment since you only have a single test for this context and the below one, but good practice to be set up for the future.

Reply:
True. For the sake of good practice, I moved it to the before :each